Doc/library/os.rst

```diff
@@ -5202,6 +5200,17 @@ Miscellaneous System Information
    .. availability:: Unix.
 
 
+.. function:: process_cpu_count()
+
+   Get the number of logical CPUs usable by the current process. Returns
```

Comment:
It is probably more accurate to say "usable by the calling thread of the current process". I believe each thread can have its own affinity.

_(For parallelism planning purposes, the main thread prior to spawning others is likely what people would be calling this from anyways)_

Reply:
Sadly, you're right. I updated the doc.